helm-charts/basehub/values.yaml

```
@@ -43,6 +43,26 @@ jupyterhub:
         projectId:
       scratchBucket:
         enabled: false
+    2i2c:
+      # Should 2i2c engineering staff user IDs be injected to the admin_users
+      # configuration of the JupyterHub's authenticator by our custom
+      # jupyterhub_config.py snippet as declared in hub.extraConfig?
+      add_staff_user_ids_to_admin_users: false
+      add_staff_user_ids_of_type: ""
```

Maybe name it as the var in the extra config? For me it wasn't clear exactly what this is until I got the extra config snippet.

```suggestion
      user_id_type: ""
```